processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -831,7 +831,7 @@ public String apply(String dimValue)
         .build();
 
     List<Row> expectedResults = Arrays.asList(
-        GroupByQueryRunnerTestHelper.createExpectedRow("2011-04-01", "alias", "", "rows", 3L, "idx", 2870L),
+        GroupByQueryRunnerTestHelper.createExpectedRow("2011-04-01", "alias", null, "rows", 3L, "idx", 2870L),
```

Comment:
this is a change in behavior and one we have to be careful about. I think it would be nice if we can preserve that empty strings are returned in place of nulls for results.